docker-for-mac/space.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Disk space
+keywords: mac, disk
+title: Disk space in Docker for Mac
+---
+
+Docker for Mac stores Linux containers and images in a single large "disk image" file
+in the Mac filesystem. This is different to Docker on Linux which usually stores containers
+and images in the `/var/lib/docker` directory.
+
+## Where is the "disk image" file?
+
+To locate the "disk image" file, first click on the whale menu and then click on
+"Preferences...". When the preferences window appears, click on the "Disk" icon
+and then "Reveal in Finder":
+
+![Disk preferences](images/settings-disk.png)
+
+The preferences window also shows how much disk space the file is currently consuming.
+In this example it is consuming only 2.4 GB out of a maximum of 64 GB.
+
+Note that other tools may display the space usage of the file incorrectly.
+
+## If the file is too big
+
+If the file is too big, you can
+- move it to a bigger drive;
+- delete unnecessary containers and images; or
+- reduce the maximum size of the file.
+
+### To move the file to a bigger drive
+
+To move the file, open the "Preferences..." menu, click on the "Disk" icon and then click
+on "Move disk image". Do not move the file directly in the finder or Docker for Mac will
+lose track of it.
+
+### To delete unnecessary containers and images
+
+To check whether you have too many unnecessary containers and images, first list images
+with:
+```bash
+$ docker image ls
+```
+then list containers with:
+```bash
+$ docker container ls -a
```

Comment:
Might want to use `docker system df` instead of checking only containers and images; doing so also shows how much the actual use is, and how much is reclaimable https://docs.docker.com/engine/reference/commandline/system_df/

Reply:
Same comment as above. Thank you for the input!

Reply:
This also looks fine to me!